source/blog/2015-02-26-digital-healthcare-roundtable-session.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: "blog"
+date: "2015-02-26 13:04:00 UTC"
+published: true
+title: "Digital Healthcare Roundtable Session"
+author: "Graeme McCubbin"
+tags:
+- Agile
+- Innovation
+---
+
+Wednesday 18th February saw the Digital Healthcare Roundtable Session take place at Unboxed Consulting.  
+
+
+ The session brought together a handpicked and diverse group of healthcare experts and leaders, with the aim to start a revolution in digital healthcare. Ideas were explored and exchanged about how we can do more to improve public health with digital technology, in this one-off discussion.  
+  
+
+ [Martyn Evans](http://www.unboxedconsulting.com/people/martyn-evans) and [Carrie Bedingfield](http://www.unboxedconsulting.com/people/carrie-bedingfield) both attended from Unboxed Consulting, with Carrie chairing the session.  
+  
+
+ ![healthcare0](http://i1291.photobucket.com/albums/b548/grammccram/DSC01779\_zpsd4fmmo16.jpg)
+
+With everyone comfortably seated, the discussion was opened, addressing the initial questions:  
+
+ • _Why is healthcare still so clunky?_  
```

Comment:
This is quite picky but there seems to be a whitespace in front of some of these lines - for example this list is not rendering as a list item: 
<img width="333" alt="screen shot 2015-09-16 at 09 54 06" src="https://cloud.githubusercontent.com/assets/885223/9900466/e7e62b42-5c58-11e5-84c2-d95222d71ed7.png">